app/components/common/Dropdown/DropdownOption/DropdownOption.tsx

```diff
@@ -44,7 +46,7 @@ const DropdownOption: React.FC<DropdownOptionProps> = ({
                 {option.icon}
               </span>
             )}
-            <span>{option.label}</span>
+            <span>{t(`${option.label}`)}</span>
```

Comment:
Why do you need backticks, `option.label` is not always a string?

Reply:
done: https://github.com/peeranha/peeranha-web/pull/1379/commits/1790d6b63600ce0c570839886c15ec8de32eba8c

Reply:
Confirm